boards/cc2538dk/include/periph_conf.h

```diff
@@ -151,6 +151,21 @@ static const periph_spi_conf_t spi_config[SPI_NUMOF] = {
 /** @} */
 
 /**
+ * @name ADC configuration
+ * @{
+ */
+#define ADC_NUMOF           (1)
+#define ADC_ALS_PIN         6
```

Comment:
I took as reference the CC2538DK documentation as default that pin is wired to the Ambient Light Sensor, but in practice it would also be required to power up the sensor via GPIO, but perhaps this should be done in another PR.  Do you have a proposal to name these pins?